lib/control.rb

```diff
@@ -0,0 +1,112 @@
+class Control
+  require_relative './plateu'
+  require_relative './rover'
+  require_relative './houston_command'
+  attr_accessor :x, :y, :plateu, :rover, :commands, :command_array
+
+  def command_rover(string)
```

Comment:
This method is the main engine - it drives the project.  But the surrounding code don't indicate that.  It's another public method in a large group of public method.  It's in a large class so it's hard to find.  Its name doesn't help show how important it is.